docs/services/Facebook.md

```diff
@@ -83,10 +83,10 @@ await FacebookService.Instance.PostToFeedWithDialogAsync(TitleText.Text, Descrip
 await FacebookService.Instance.PostToFeedAsync(TitleText.Text, MessageText.Text, DescriptionText.Text, picture.Name, stream);
 
 // Get current user's photo albums
-await FacebookService.Instance.GetUserAlbumsAsync();
+await FacebookService.Instance.GetUserAlbumsAsync(20, FacebookAlbum.Fields);
```

Comment:
we have a method with original signature.. we should continue using that.. no need to change that here